n2v/grid/grider.py

```diff
@@ -455,6 +465,12 @@ def on_grid_esp(self, grid=None, vpot=None, wfn=None):
         hartree = - 1.0 * (vext + esp)
         v_fa = (1 - 1.0 / (self.nalpha + self.nbeta)) * hartree
 
+        if Da is not None:
+            self.wfn.Da().np[:] = Da_temp
+            print(np.linalg.norm(Da_temp))
```

Comment:
Why are we printing the norm? Was this used for testing?

Reply:
Yes I will delete that.